src/client/features/view/layout/coseBilkent.js

```diff
@@ -1,13 +0,0 @@
-const coseBilkentOpts = {
-  name: 'cose-bilkent',
-  fit: true,
```

Comment:
Did you still need these layout files?

Reply:
They were redundant. I must have quickly copied them at some point, but they exist elsewhere. I just changed the reference and deleted the copies.

Reply:
Oh okay.